apis/datadoghq/v1alpha1/utils.go

```diff
@@ -27,6 +27,13 @@ func GetConfName(owner metav1.Object, conf *CustomConfigSpec, defaultName string
 	return fmt.Sprintf("%s-%s", owner.GetName(), defaultName)
 }
 
+// GetAgentServiceAccount returns the node agent serviceAccountName
+func GetAgentServiceAccount(dda *DatadogAgent) string {
```

Comment:
i noticed there is a similar method https://github.com/DataDog/datadog-operator/blob/c07038fbf0b8993798ec5693cf6f191d7c369feb/controllers/datadogagent/utils.go#L1833

(that file also seems to have duplicates of `GetClusterAgentServiceAccount` and `GetClusterChecksRunnerServiceaccount`)

Reply:
I tried using the controller utils and that caused a circular import error with go, probably from [importing eventcollection](https://github.com/DataDog/datadog-operator/pull/500/files#diff-827f2f59a3c51c89207aae6aeab23a903d8451f0f9123064bd71582b14e527abR38) in the `controller.go` file. I left it as is for now, but would be open to moving around those utility functions if that would be better.